Zadanie 0

wsparcie opened the pull request:
Rozwiązanie do zadania 0 - Elo Żelo.

wsparcie committed:
feat: add index.ts with eloZelo function to write 'Elo żelo' to .txt file n times
- n depends on the number of minutes in the current time

wsparcie committed:
feat: add iterator to console logs

czaja307 commented on index.ts:
```diff
@@ -0,0 +1,15 @@
+  const file: string = './elo-żelo.txt'
+  writeFileSync(file, '')
+  for (let i: number = 0; i < repeats; i++) {
+    writeFileSync(file, text + '\n', { flag: 'a' })
```
disc operations (e.g. fileWrite) are quite time-consuming - try to prepare the string first and the write the file using only one `writeFileSync` call

czaja307 commented on index.ts:
```diff
@@ -0,0 +1,15 @@
+}
+
+let minutes: number = new Date().getMinutes()
+console.log(`minutes: ${minutes}`)
```
console logs are a great tool for debugging, but try to avoid leaving them in the final code - they slow down the execution, but what's more important they might leak some sensitive informations

czaja307 reviewed (changes requested):
just few minor issues, but you got the idea right :D

wsparcie committed:
refactor:
- use single writeFileSync with previously created contents
- remove console.logs

czaja307 reviewed (approved):
🔥
